### PR TITLE
Added centralized logging via ELK

### PR DESCRIPTION
**Pull request by Castyr**

### CODE REVIEW

**Comment by Castyr** [comment]
[Bank: Create logging aggregator for docker](https://trello.com/c/IWuWB4UZ/131-bank-create-logging-aggregator-for-docker)

**Comment by Castyr** [comment]
[Bank: Create logging aggregator for docker](https://trello.com/c/IWuWB4UZ/131-bank-create-logging-aggregator-for-docker)

**Comment by Castyr** [comment]
[Bank: Create logging aggregator for docker](https://trello.com/c/IWuWB4UZ/131-bank-create-logging-aggregator-for-docker)

File: account-query/out/production/resources/application.yml [hunk 1]
```diff
@@ -0,0 +1,32 @@
+# Mongo config
```
Comment (R4il) [inline account-query/out/production/resources/application.yml]: This guy is  back at it again...

File: elk/docker-compose.yml [hunk 2]
```diff
@@ -0,0 +1,48 @@
+version: '2'
```
Comment (R4il) [inline elk/docker-compose.yml]: Does this one has to be v2 too? I dont see mem limit in use

File: transactions/build.gradle [hunk 3]
```diff
@@ -1,9 +1,6 @@
 dependencies {
-    compile('org.springframework.cloud:spring-cloud-starter-netflix-eureka-client')
```
Comment (R4il) [inline transactions/build.gradle]: if transactions uses accounts services wont it need eureka?

**Pull request merged**